src/components/FileDetail/FileDetail.js

```diff
@@ -0,0 +1,207 @@
+                          kind={fileTypeDetail[fileNode.fileType].icon}
+                        />
+                      </div>
+                      <p className="FileInfo--Text">
+                        {fileTypeDetail[fileNode.fileType].title}
+                      </p>
+                    </div>
+                  </div>
+                  <div className="cell-6 sm:cell-3">
+                    <p className="FileInfo--Title">Updated:</p>
+                    {latestDate ? (
+                      <div className="flex">
+                        <p className="FileInfo--Text">
+                          <TimeAgo
+                            date={latestDate}
+                            live={false}
+                            className="mr-4"
+                          />
+                          by
+                        </p>
+                        <Avatar
+                          className="inline-block ml-4 mt-4"
+                          size={20}
+                          imgUrl="https://www.w3schools.com/css/img_avatar.png"
+                        />
+                      </div>
+                    ) : (
+                      <p className="FileInfo--Text">Unknown</p>
+                    )}
+                  </div>
+                  <div className="cell-6 sm:cell-2">
+                    <p className="FileInfo--Title">Size:</p>
+                    <p className="FileInfo--Text">{latestSize}</p>
+                  </div>
+                </GridContainer>
+              </div>
+
+              <div className="row-3 md:row-2 lg:cell-2 md:cell-3 cell-12 flex flex-row items-start h-full w-full justify-end flex-wrap">
+                <Link
+                  to={`/study/${match.params.kfId}/files/${
+                    fileNode.kfId
+                  }/annotation/`}
+                  className="no-underline"
+                >
+                  <Button type="button" icon="edit">
+                    EDIT
+                  </Button>
+                </Link>
+                <button
+                  className="text-grey test-xs font-light underline ml-12 my-4"
+                  onClick={e => {
+                    deleteFile({variables: {kfId: fileNode.kfId}});
+                    history.goBack();
+                  }}
+                >
+                  <Icon className="mr-4" width={10} height={10} kind="delete" />
+                  <span>Delete</span>
+                </button>
+                <Button
+                  type="button"
+                  icon="download"
+                  color="primary"
+                  onClick={e => {
+                    downloadFile({
+                      variables: {
+                        studyId,
+                        fileId: fileNode.kfId,
+                      },
+                    }).then(resp => {
+                      const url = `${KF_STUDY_API}${resp.data.signedUrl.url}`;
+                      window.location.href = url;
```

Comment:
try using `history.push` here so that it keeps track of url changes in the react-router